docs/src/pages/Homepage.vue

```diff
@@ -53,6 +73,83 @@
 import { defineComponent } from 'vue'
 import { useMeta } from 'quasar'
 import { useDocStore } from 'assets/doc-store.js'
+import {
+  mdiAnimation,
+  mdiApplicationExport,
+  mdiCalendar,
+  mdiCardMultiple,
+  mdiFormDropdown,
+  mdiFormTextbox,
+  mdiImageSizeSelectSmall,
+  mdiTable
+} from '@quasar/extras/mdi-v6'
+
+const importantPages = {
```

Comment:
Avoid nested models when not needed: use two `const` for the 2 arrays

Reply:
Unless you use the `importantPages` into a v-for to avoid duplication, which is not the case here for now